docker/z-quantum-default/Dockerfile

```diff
@@ -23,19 +26,22 @@ RUN apt-get -y install \
                 libzmq3-dev \
                 libz-dev \
                 libblas-dev \
-                liblapack-dev
+                liblapack-dev \
+                openssh-client
 
 
-# ZQuantum requires Python 3.7. By installing `python3-pip` we have also installed 
-# Python 3.8. We need to set back the Python version to 3.7.
-RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1 && \
-    update-alternatives --set python3 /usr/bin/python3.7
+#- # After March 2022, git changed its authentication.
```

Comment:
I couldn't understand what this comment is about.

Reply:
Will remove. It was about how the old dockerfile was trying to clone. Its not really helpful now.